doc/contributor/cexts.md

```diff
@@ -32,19 +40,29 @@ jt gem-test-pack
 You can then test C extension support.
 
 ```bash
-GEM_HOME=$(jt gem-test-pack)/gems jt test cexts --no-libxml --no-openssl
+jt test cexts --no-libxml
```

Comment:
Don't we need the gem-test-pack for testing cexts?
Or is it handled by `jt` automatically? (I would prefer that)